matplotlib_pyodide/browser_backend.py

```diff
@@ -190,6 +190,13 @@ def ignore(event):
 
         self.draw()
 
+    def destroy(self, *args, **kwargs):
+        div = document.getElementById(self._id)
```

Comment:
Please handle the case if the return value of document.getElementById is null (when the element id does not exist for any reason).

Reply:
As the picture below, the self._id attribute is init by class FigureCanvasWasm, which is always avaliable, so I don't need to write redundant null logic.
![image](https://github.com/pyodide/matplotlib-pyodide/assets/111437058/1607a308-0e1c-4758-b98e-c398f9521eb9)

Reply:
> As the picture below, the self._id attribute is init by class FigureCanvasWasm, which is always avaliable, so I don't need to write redundant null logic.

Actually, this is not always true because users can remove HTML elements manually for any reason. But maybe we can say that it is the user's fault.